packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/ScatterPlotViewer.stories.js

```diff
@@ -58,9 +65,17 @@ const mockStore = {
   }
 }
 
-function ViewerContext(props) {
-  const { children } = props
-  return <Provider classifierStore={mockStore}>{children}</Provider>
+function ViewerContext({
+  subject = transientObjectSubject,
+  children
+}) {
+  const classifierStore = useMemo(() => ({
+    ...mockStore,
+    subjects: {
+      active: subject
+    }
+  }), [mockStore, subject])
```

Comment:
It might be safer to give each story its own store, rather than generate a store object on each render. The latter can generate errors in `mobx-react`, which expects stores to be final.